org.catacombing.kumo.json

```diff
@@ -0,0 +1,126 @@
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_C_FLAGS_RELEASE=-DNDEBUG",
+                "-DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/google/woff2/archive/v1.0.2.tar.gz",
+                    "sha256": "add272bb09e6384a4833ffca4896350fdb16e0ca22df68c0384773c67a175594"
+                }
+            ]
+        },
+        {
+            "name": "libwpe",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_C_FLAGS_RELEASE=-DNDEBUG",
+                "-DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://wpewebkit.org/releases/libwpe-1.16.2.tar.xz",
+                    "sha256": "960bdd11c3f2cf5bd91569603ed6d2aa42fd4000ed7cac930a804eac367888d7"
+                }
+            ]
+        },
+        {
+            "name": "wpewebkit",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DPORT=WPE",
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_C_FLAGS_RELEASE=-DNDEBUG",
+                "-DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG",
+                "-DUSE_SKIA_OPENTYPE_SVG=OFF",
+                "-DUSE_SYSPROF_CAPTURE=OFF",
+                "-DUSE_LIBBACKTRACE=OFF",
+                "-DENABLE_WPE_PLATFORM=ON",
+                "-DENABLE_BUBBLEWRAP_SANDBOX=OFF",
+                "-DENABLE_WPE_PLATFORM_DRM=OFF",
+                "-DENABLE_SPEECH_SYNTHESIS=OFF",
+                "-DENABLE_DOCUMENTATION=OFF",
+                "-DENABLE_MINIBROWSER=OFF"
+            ],
+            "build-options": {
+                "cflags": "-g1",
+                "cxxflags": "-g1"
+            },
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/chrisduerr/WebKit",
+                    "commit": "937c2d0607cc115f331eda107cde18283afb0f5c"
+                }
+            ]
+        },
+        {
+            "name": "kumo",
+            "buildsystem": "simple",
+            "build-commands": [
+                "cargo --offline fetch",
+                "cargo --offline build --release",
+                "install -Dm 755 ./target/release/kumo /app/bin/kumo",
+                "install -Dm 644 ./logo.svg /app/share/icons/hicolor/scalable/apps/org.catacombing.kumo.svg",
+                "desktop-file-edit --set-name=org.catacombing.kumo Kumo.desktop",
+                "desktop-file-edit --set-icon=org.catacombing.kumo Kumo.desktop",
+                "install -Dm 644 ./Kumo.desktop /app/share/applications/org.catacombing.kumo.desktop",
+                "install -Dm 644 ./org.catacombing.kumo.metainfo.xml -t /app/share/metainfo/"
```

Comment:
if this is mobile only it should be marked in metainfo as such

![image](https://github.com/user-attachments/assets/b050481b-3cca-4be2-8d28-59c8bc2eec09)

https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines#mobile-only-apps

Reply:
> if this is mobile only it should be marked in metainfo as such

It technically supports non-mobile environments. It just works *better* on mobile and is optimized for it. I'm not sure that qualifies as "only on mobile"?

Reply:
> It technically supports non-mobile environments. 

the question is whether you do / intended platform and if it is usable.